provider/blob/src/blob_schema.rs

```diff
@@ -24,16 +24,20 @@ pub(crate) struct BlobSchemaV1<'data> {
     /// Weak invariant: there is at least one value for every integer in 0..self.buffers.len()
     #[serde(borrow)]
     pub keys: ZeroMap2dBorrowed<'data, DataKeyHash, Index32U8, usize>,
-    /// Vector of buffers
+    /// Indices into the buffer slice
     #[serde(borrow)]
-    pub buffers: &'data VarZeroSlice<[u8], Index32>,
+    pub indices: &'data FlexZeroSlice,
```

Comment:
Note: I could instead write this to have the exact same binary representation as the VarZeroSlice, which would allow us to make the change later and go back and forth.

Reply:
If there is interest, this could be `UnvalidatedVarZeroVec`. It would support only one type, byte slices, and the indices would be lazily validated.